README.md

```diff
@@ -1,10 +1,74 @@
+    The innovative and efficient way to market your farm products.
+    <br />
+    <a href="https://cubeseed.com.ng"><strong>Learn more »</strong></a>
+    <br />
+    <br />
+    <a href="https://cubeseed.com.ng">Website</a>
+    ·
+    <a href="https://discord.gg/JPBmEcKmpY">Discord</a>
+    ·
+    <a href="https://github.com/cubeseed/Frontend/issues">Issues</a>
+  </p>
+</p>
+
+<p align="center">
+   <a href="https://discord.gg/JPBmEcKmpY"><img src="https://img.shields.io/badge/Discord-discord.gg%2FCubeseed-%234A154" alt="Join Cubeseed Discord"></a>
+   <a href="https://github.com/cubeseed/Frontend/stargazers"><img src="https://img.shields.io/github/stars/cubeseed/Frontend" alt="Github Stars"></a>
+   <a href="https://github.com/cubeseed/Frontend/pulse"><img src="https://img.shields.io/github/commit-activity/m/cubeseed/Frontend" alt="Commits-per-month"></a>
+   <a href="https://github.com/cubeseed/Frontend/issues?q=is:issue+is:open+label:%22%F0%9F%99%8B%F0%9F%8F%BB%E2%80%8D%E2%99%82%EF%B8%8Fhelp+wanted%22"><img src="https://img.shields.io/badge/Help%20Wanted-Contribute-blue"></a>
+</p>
+
+## About the Project
+
+
+<img width="100%" alt="booking-screen" src="https://github.com/KrownWealth/Frontend/assets/25585850/65307db6-f1ea-4376-869b-5c2e0d450592">
+
+# Seamless Agro-Marketing with Cubeseed: Connecting Farmers and Buyers
+
+Cubeseed helps small farmers in Africa access quality inputs, markets, and finance using AI. Our tool helps farmers trade safely, quickly, and profitably. We want to fix the problem of inefficient and unsafe agricultural trading in Africa.
+
+Cubeseed is a cutting-edge contract-driven marketplace fueled by AI, designed to facilitate seamless agro-trade and connect smallholder farmers to quality inputs, markets, and finance. With Cubeseed, farmers can trade securely, swiftly, and profitably, empowering them to achieve uninterrupted farm operations. Our ultimate goal is to enhance agricultural production significantly, addressing pressing issues such as food insecurity, unemployment, and high food costs.
+
+With Cubeseed, connecting with processors, input producers, and service providers has been streamlined and made accessible like never before. Our revolutionary application harnesses the power of AI and a contract-driven marketplace to facilitate direct connections between farmers and these essential stakeholders.
+
+# Overview
+
+This repository contains the Frontend development workflow for CubeSeed.
+
+## Tech Stack
+ * [Next JS](https://nextjs.org/ )
+ * [Typescript](https://www.typescriptlang.org/)
+ * [Tailwind CSS](https://tailwindcss.com/)
+ * [Saas](https://sass-lang.com/)
+ * [React Query](https://tanstack.com/query/v3/)
+ * [Contect API](https://react.dev/reference/react/useContext)
+ * [Farmer motion or Gsap for animation](https://www.framer.com/motion/)
+ * [Node JS](https://nodejs.org/en/docs)
+ * [Express JS](https://expressjs.com/en/guide/routing.html)
+ * [SQL](https://sql-academy.org/en)
```

Comment:
I think there should not be SQL in the front end (in the backend either)

Reply:
You're right, SQL is a kind of database stuff.

I will do the edit now.